mtp_noms_ops/assets-src/stylesheets/views/_checks.scss

```diff
@@ -73,3 +73,84 @@ table.mtp-table--review {
 .button-warning {
   @include button($red);
 }
+
+// TODO remove once new gov.uk styling implemented
+.moj-sub-navigation {
+  margin-bottom: 0.7rem;
+}
+
+
+.moj-sub-navigation__list {
+  font-size: 0; // Removes white space when using inline-block on child element.
+  box-shadow: inset 0 -1px 0 #b1b4b6;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+
+}
+
+
+.moj-sub-navigation__item {
+  line-height: 1.31579;
+  display: inline-block;
+
+  font-family: "Helvetica Neue", "Inter UI", sans-serif;
+  font-size: 1.1875rem;
+  font-weight: 400;
+
+  margin-top: -1px;
+  margin-right: 20px;
+
+  &:last-child {
+    box-shadow: none;
+  }
+}
+
+
+.moj-sub-navigation__link {
+  display: block;
+  padding-top: 12px;
+  padding-bottom: 12px;
+  padding-left: 0.3rem;
+  text-decoration: none;
+  position: relative;
+
+  &:focus {
+    color: black; // Focus colour on yellow should really be black.
+    position: relative; // Ensure focus sits above everything else.
+    box-shadow: none;
+    outline: 3px solid transparent;
+    background-color: #ffdd00;
```

Comment:
we should use `$focus-colour` to pick up the old-style focus colour (it's a different yellow)